### PR TITLE
Review Request on Login & Register removal from Navbar

### LINKED ISSUE #77: Move Login and Register buttons to landing page

### PR DESCRIPTION
**Pull request by Nejo12**
Screenshot 2021-01-05 at 15 01 38

* Login & Register removed from Navbar to Landing Page.
* LogOut component dynamically implemented into NavBar - Team's contribution is required.
* Styles (SCSS) on NavBar cleaned up - a bit :persevere:

### CODE REVIEW

**Comment by dilippoudel** [comment]
Could you write a code to show the logout button conditionally if the user
is logged in?

**Comment by Nejo12** [comment]
Yes, I could, but i could not figure out what the response is from the backend. I thereby included a temporary option as seen in the attached screenshot below:
Screenshot 2021-01-05 at 16 18 28

Btw, is browser's localStorage being used? Because there was nothing there after registration or login in. 
Kindly enlighten me on the misunderstood procedure.

File: src/components/NavBar/index.tsx [hunk 1]
```diff
@@ -1,28 +1,20 @@
 import React from 'react'
-import { LinkContainer } from 'react-router-bootstrap'
-import { Navbar, Nav } from 'react-bootstrap'
+import { Nav, Navbar } from 'react-bootstrap'
+import LogOut from '../LogOut'
 
 const NavBar = () => {
+  const isAuthenticated = true // temporarily included. Will be after confirmation of authentication from backend team.
```
Comment (Wyfy0107) [inline src/components/NavBar/index.tsx:12]: This should be stored inside `redux`. Perhaps you can make a `login` state that has all the related information about credential stuffs

**Review by Wyfy0107 — CHANGES_REQUESTED** [review]
Looks awesome. You can either add the `login` state in this pr or remove it and open another one to include `login` state

**Comment by Tsaikkari** [comment]
Shouldn't the navbar on top be shown on the other pages only, not on the landing page? Otherwise, what was the point to remove the login and register into the landing page and keep the navbar anyway? :))))

**Comment by Tsaikkari** [comment]
I think we shouldn't have buttons or links that don't have any purpose to be there, like the logout button on the navbar on the landing page.

**Comment by Wyfy0107** [comment]
> Shouldn't the navbar on top be shown on the other pages only, not on the landing page? Otherwise, what was the point to remove the login and register into the landing page and keep the navbar anyway? :))))

This is valid, let's discuss more in next meeting !

**Comment by Nejo12** [comment]
Screenshot 2021-01-05 at 20 41 33

NavBar removed from Landing Page - team agreement / suggestion required prior to merging.

**Comment by dilippoudel** [comment]
I like this design✅

**Review by Wyfy0107 — APPROVED** [review]

**Comment by Wyfy0107** [comment]
You can merge it after resolving the conflicts 🥇

**Comment by Nejo12** [comment]
> You can merge it after resolving the conflicts 🥇

It's weird that `.eslintcache` is deleted without touching it

**Comment by Wyfy0107** [comment]
> > You can merge it after resolving the conflicts 🥇
> 
> It's weird that `.eslintcache` is deleted without touching it

I think @Tsaikkari deleted it in her PR. This file is annoying.

**Pull request merged**